tests/e2e/specs/global/tools.spec.prod.js

```diff
@@ -1,12 +1,21 @@
 function setup() {
   cy.wipeDatabase();
-  cy.visit('/');
+  cy.visit('/?mock-prod=true', {
```

Comment:
Added a query param to allow us to mock a "true prod" environment. I know this is basically a shim but it's at least better coverage than none.